### PR TITLE
feature: filter groups by environment for batch operations

### PR DESCRIPTION
**Pull request by aaronkvanmeerten**
Force environment to be sent when doing batch operations on groups

### CODE REVIEW

File: src/instance_group.ts [hunk 1]
```diff
@@ -132,15 +132,22 @@ export default class InstanceGroupManager {
         ctx: Context,
         type: string,
         region: string,
+        environment: string,
     ): Promise<Array<InstanceGroup>> {
         const groups = await this.getAllInstanceGroups(ctx);
 
         function byTypeAndRegion(group: InstanceGroup) {
```
Comment (raluca8x8) [inline src/instance_group.ts:139]: Please rename byTypeAndRegion to something similar to byTypeRegionAndEnv. The same for getAllInstanceGroupsByTypeAndRegion.

File: src/scaling_options_manager.ts [hunk 2]
```diff
@@ -24,13 +24,14 @@ export default class ScalingManager {
         ctx: Context,
     ): Promise<FullScalingOptionsResponse> {
         ctx.logger.info(
-            `Updating scaling options for groups of type ${request.instanceType} in region ${request.region}, scaling ${request.direction}`,
+            `Updating scaling options for groups environment ${request.environment} of type ${request.instanceType} in region ${request.region}, scaling ${request.direction}`,
```
Comment (raluca8x8) [inline src/scaling_options_manager.ts:27]: Please add the environment to these log messages as well:

`No groups of type ${request.instanceType} were found to update in region ${request.region}.
`Groups of type ${request.instanceType} from region ${request.region} are now updated with the new scaling options`,

**Review by oanaianc — APPROVED** [review]

File: src/instance_group.ts (hunk 1 above)
Comment (aaronkvanmeerten) [reply src/instance_group.ts:139]: Done

File: src/scaling_options_manager.ts (hunk 2 above)
Comment (aaronkvanmeerten) [reply src/scaling_options_manager.ts:27]: done

**Review by sawall — APPROVED** [review]

**Review by raluca8x8 — APPROVED** [review]

**Pull request merged**